src/content.rs

```diff
@@ -434,6 +422,63 @@ impl FromStr for Content {
     }
 }
 
+type ProcessFileReturn = (BTreeMap<String, PageValues>, Option<DateTime<Utc>>);
+
+fn process_files(
+    files: Vec<PathBuf>,
+    show_unpublished: bool,
+) -> Result<ProcessFileReturn, anyhow::Error> {
```

Comment:
Looking at how this is used it seems like this could just return an IndexCache?  I'm probably missing something though.

Reply:
good call!